### PR TITLE
Update the site with more information about developing.

### PR DESCRIPTION
**Pull request by omalley**

### CODE REVIEW

**Comment by omalley** [comment]
I moved the committer list to a new page, because it made all of the other information get lost below the long list.

I added sections on how to set up git, approve pull requests, create a gpg key, and the dist directory.

I also added Xiening Dai and Gang Wu as committers.

**Comment by xndai** [comment]
LGTM. Thanks for putting together a document.

File: site/develop/committers.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Ferdinand Xu            | xuf          | Committer
+Yongzhi Chen            | ychena       | Committer
+Aihua Xu                | zihuaxu      | Committer
+
+Companies with employees that are committers:
+
+* Alibaba
+* Cloudera
+* Facebook
+* Hewlett Packard Enterprise
```
Comment (majetideepak) [inline site/develop/committers.md]: Can we add Vertica instead of HPE?

Comment (omalley) [reply site/develop/committers.md]: Done

**Pull request closed without merging**